scope.py

```diff
@@ -152,23 +155,24 @@ def _get_nearest_gaia(
                             "phot_rp_mean_mag": 1,
                             "ra": 1,
                             "dec": 1,
-                        }
+                        },
                     }
-                }
+                },
             },
-            "kwargs": {
-                "limit": 1
-            }
+            "kwargs": {"limit": 1},
         }
         response = self.kowalski.query(query=query)
         gaia_nearest = [
-            v[0] for k, v in response.get("data").get(catalog).items()
-            if len(v) > 0
+            v[0] for k, v in response.get("data").get(catalog).items() if len(v) > 0
         ]
         df = pd.DataFrame.from_records(gaia_nearest)
 
         df["M"] = df["phot_g_mean_mag"] + 5 * np.log10(df["parallax"] * 0.001) + 5
-        df["Ml"] = df["phot_g_mean_mag"] + 5 * np.log10((df["parallax"] + df["parallax_error"]) * 0.001) + 5
+        df["Ml"] = (
+            df["phot_g_mean_mag"]
+            + 5 * np.log10((df["parallax"] + df["parallax_error"]) * 0.001)
+            + 5
+        )
         df["BP-RP"] = df["phot_bp_mean_mag"] - df["phot_rp_mean_mag"]
```

Comment:
Are we using errors on BP/RP in any way? The question that was asked of Laurent in the first ZTF-Gaia talk was important: will they be presenting average colors with error-bars in DR3. He didn't know.

Reply:
This part of code wasn't touched in this PR, except for linting, so I think this should be moved into issues.
(I think we don't use it in this context).